src/controllers.lisp

```diff
@@ -16,11 +16,13 @@
 (defmethod on-click (instance x y))
 (defmethod on-middle-click (instance x y))
 (defmethod on-right-click (instance x y))
+(defmethod on-click-release (instance x y))
+(defmethod on-middle-click-release (instance x y))
```

Comment:
are the new methods not using `with-sketch` / `draw-mode` ?

probably a good candidate for a file-local macro..but we can do that later

Reply:
I think I forgot to add that.

Reply:
And yes, a macro would be nice to remove all the repetitive code.